pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: facebook_app_events
 description: Flutter plugin for Facebook App Events, an app measurement
   solution that provides insight on app usage and user engagement in Facebook Analytics.
-version: 0.11.0
+version: 0.2.0
```

Comment:
Version `0.2.0` is **lower** than `0.11.0` 

And also please refrain from bumping version in feature PR since there might be releases in between the time it takes for this PR to be merged.

Please revert this line